website/snippets/_cloud-environments-info.md

```diff
@@ -42,3 +42,48 @@ By default, all environments will use the default branch in your repository (usu
 - **Deployment:** determines the branch is cloned during job executions for each environment.
 
 For more info, check out this [FAQ page on this topic](/faqs/Environments/custom-branch-settings)!
+
+
+### Extended attributes (Beta)
+
+:::important This feature is currently in beta
+
+Extended Attributes allow users to enter YAML attributes in their Environment settings. It's currently in beta and if you're interested in joining the beta, please contact us -- HOW CAN USES TRY THIS BETA? IF IT'S FEATURE FLAG, HOW DO THEY JOIN THE BETA? DO WE NEED TO ADD A FORM LINK? OR EMAIL?
+:::
```

Comment:
It is a feature-flag enabled beta, users cannot opt in via dbt Experiments. This is because it is an account-level feature. 

I will defer to @Fleid, but the following is my understanding:

We will have a very small, targeted beta -- primarily existing customers who we know will benefit from this (blue shield & a few more). So I don't know that we're doing an open "call for beta" - the hope would be to get it into GA ASAP.

So we may be able to exclude any mention of joining the beta?